### ISSUE TITLE
Needs updating to assure Pandas v.0.23 compatibility

### ISSUE THREAD

**Issue by davidsvale** [UDST/urbanaccess#60]
#### Description of the bug
Pandas v. 0.23.0 (released on May 15, 2018) has deprecated `data frame.as_matrix()`, used when creating the integrated transit and pedestrian network.

It is used in `_nearest_neighbor(df1,df2)` within `network.py`

It should be replaced by `dataframe.values`

#### GTFS feed or OSM data (optional) 
If the issue is related to a specific GTFS feed or OSM data please provide the URL to download the GTFS feed or the bounding box used to extract the OSM data.

#### Environment 

- Operating system: MacOS

- Python version: 3.7.0

- UrbanAccess version: 0.2.0

- UrbanAccess required packages versions (optional):

#### Paste the code that reproduces the issue here:

```python
# ua.network.integrate_network(urbanaccess_network=urbanaccess_net, headways=False)
```

#### Paste the error message (if applicable):
```python
# AttributeError: 'DataFrame' object has no attribute 'as_matrix'
```

**Comment by sablanchard** [comment]
Thanks @davidsvale ! We will take a look soon and plan to address this for the next release.

**Comment by smmaurer** [comment]
Resolved in PR #65, release coming soon.
